### ISSUE TITLE
Change font colour

### ISSUE THREAD

**Issue by calvinbui** [flixlix/power-flow-card-plus#33]
It'll be nice if the font colour can be changed.

My solar and home text are white while the grid is coloured.

![image](https://user-images.githubusercontent.com/3604363/228084321-2417911a-d09c-4a8c-aa4c-22ada095f6b2.png)

Example with card-mod:

![image](https://user-images.githubusercontent.com/3604363/228084571-4016186b-dcf1-40b1-be99-16d13fd135f9.png)

**Comment by flixlix** [comment]
I'll add this to my to-do list 👍

**Comment by flixlix** [comment]
Added this feature in the latest release! 🎉

**Comment by calvinbui** [comment]
@flixlix I can't get the text to be red when setting `color_value: true`. setting `color_value` to `solar` or `grid` worked however.

![image](https://user-images.githubusercontent.com/3604363/231487609-f7e7d968-153d-4f29-9960-549f97ee48b4.png)

```
    view_layout:
      grid-area: flow
    type: custom:power-flow-card-plus
    clickable_entities: true
    kw_decimals: 2
    max_expected_flow_w: 10000
    entities:
      home:
        entity: sensor.home_power_consumption
        color_icon: true
        color_value: true
      grid:
        entity:
          consumption: sensor.grid_power_consumption
          production: sensor.grid_power_export
        color_icon: true
        color:
          production: '#18cf87'
          consumption: '#f76354'
        display_state: two_way
        color_circle: true
      solar:
        entity: sensor.total_solar_generation
        color_icon: true
```

**Comment by flixlix** [comment]
I'm sorry, I forgot to correct for the value...
Fixed in #68
